config.yaml

```diff
@@ -1,25 +1,25 @@
 watches: 
   - flavor: "baremetal"
-    fileName: "sl-micro-60-baremetal"
-    displayName: "SL Micro 6.0"
-    osRepo: registry.suse.com/suse/sl-micro/6.0/baremetal-os-container
-    isoRepo: registry.suse.com/suse/sl-micro/6.0/baremetal-iso-image
-    limit: 3
+    fileName: "sl-micro-61-baremetal"
+    displayName: "SL Micro 6.1"
+    osRepo: registry.suse.com/suse/sl-micro/6.1/baremetal-os-container
+    isoRepo: registry.suse.com/suse/sl-micro/6.1/baremetal-iso-image
+    limit: 1
```

Comment:
Only one because we want to publish GM and once this is out keep adding on top, but not pre-GM